src/microbe_stage/MicrobeColony.cs

```diff
@@ -71,6 +71,24 @@ public void RemoveFromColony(Microbe microbe)
         if (State == Microbe.MicrobeState.Unbinding)
             State = Microbe.MicrobeState.Normal;
 
+        var rotation = microbe.RotationInsideColony();
+        var offset = microbe.GetOffsetRelativeToMaster();
+
+        // Apply the rotation
+        if (microbe != Master)
+        {
+            // For some reason the first cell needs to be rotated by the colony master rotation
+            if (microbe.ColonyParent == Master)
+                rotation *= new Quat(Master.Transform.basis);
+
+            // Normalize and apply the rotation
+            rotation = rotation.Normalized();
+            microbe.LookAtPoint = rotation.Xform(offset);
```

Comment:
Why is offset used here? Wouldn't a vector of just 1, 0, 0 or something similar work?

Reply:
The cell might be bigger then that vector and so it could point in the wrong way, if we use the distance between it and the master we know for sure that the position will be outside the cell.

Reply:
The look at point *doesn't* need to be outside the cell, the rotation direction is calculated from the vector (look at position - cell center position). So it doesn't matter as long as the point is not *exactly* at the cell center. In fact adding the position on the colony *could* accidentally make the point match exactly the world position the microbe is detached at.